### ISSUE TITLE
ISO 19135 definition of "registration"

### ISSUE THREAD

**Issue by heidivanparys** [ISO-TC211/StandardsTracker#397]
In ISO 19135, "registration" is currently defined as "assignment of a permanent, unique, and unambiguous identifier to an item". Is identifier assignment really the essential characteristic of "registration"? I agree that in many cases identifier assignment will indeed be done, but "registration" can also these days still happen in an analogue way. E.g. the registration of a guest on a [guest list](https://www.qwant.com/?t=images&q=%22guest+list%22+template). In many cases, such kinds of lists will not have a field with a permanent, unique, and unambiguous identifier.

So I would argue that the current definition is too narrow and also against the principle that says that [definitions must be application neutral](https://arkitektur.digst.dk/en/metoder/regler-begrebs-og-datamodellering/compose-application-neutral-definitions) (meaning I cannot reuse this definition in a Danish context and have to use another one or make up a new one).

Another argument for the need for a change of definition, is that the current definition is too far from what common language dictionaries say, whereas it seems that the meaning is actually the same, when you read the standard.

E.g.:

- the act of making an official record of something/somebody ([Oxford Learner's Dictionaries](https://www.oxfordlearnersdictionaries.com/definition/english/registration?q=registration))
- The registration of something such as a person's name or the details of an event is the recording of it in an official list. ([Collins Dictionary](https://www.collinsdictionary.com/dictionary/english/registration))
- The act of registering (and register = to make or secure official entry of in a register) ([Merriam-Webster's Dictionary](https://www.merriam-webster.com/dictionary/registration), https://www.merriam-webster.com/dictionary/registering)

In the context where I needed the concept "registration", I used the following:

> **registration**
> the act of entering something into an information system
> 
> Note 1 to entry: Registration implies that what is registered persists. In other words, what is registered becomes persistent, it still exists even when the registration is completed.
> 
> Note 2 to entry: Often, a very significant part of a registration is to assign a permanent, unique and unambiguous identifier to what is registered.

So the information about the identifier is moved from the definition to a note.

The reason the definition uses "information system" and not "register", is that we tried to avoid that word, as there has been a lot of confusion regarding register vs. registry, and the different meanings of "register". That will be hopefully solved in the revision of ISO 19135.

**Comment by PeterParslow** [comment]
I have often thought that this definition in 19135 is rather terse, to the point of being effectively meaningless - but that seems to be common to many definitions in standards!

I think that analogue registration is out of scope, given that the scope is explicitly "geographic information"

The background to this definition is most likely the ISO Directives (https://www.iso.org/sites/directives/current/consolidated/index.xhtml#_idTextAnchor415), which uses the phrase "Registration elements" which are defined as "Unique identifiers or identifier code components".

Note also that if we (Ron's project) changes the definition of 'registration' then that implicitly changes the scope of the standard, which currently reads "This part of ISO 19135 specifies procedures to be followed in establishing, maintaining, and publishing
registers of unique, unambiguous, and permanent identifiers and meanings that are assigned to items of
geographic information."

It would also have a knock on effect on the definition of "register", currently "set of files containing _identifiers_ assigned to items with descriptions of the associated items"

Perhaps what we need is to bring into the definition that other aspect "and meanings", which is currently implicit in "items"? But we need to be careful not to make ISO 19135 too prescriptive about what information the various different registers would need to hold - that's the job of the defining document of that register (for ISO ones, the "RA standard" (e.g. ISO 19127) or "standard that will require frequent modification" (status being considered for ISO 19157-3)

**Comment by heidivanparys** [comment]
> [...] to the point of being effectively meaningless - but that seems to be common to many definitions in standards!

Let's try to change that in future standards!

As always, another option than changing the definition is to narrow down the term so it better matches the intended meaning ("item registration"? "digital item registration"?) or to add a domain in front of the register (many definitions in this standard have "register" in front of the definition. I don't have a final solution, I just hope it can be discussed in the upcoming revision, so we can make our terminology as useful and reusable as possible.

Regarding the definition of "register", that is worth another issue... 🙂

**Comment by PeterParslow** [comment]
The revision of ISO 19135-1 includes "Resolve the confusion between “register” and “registry”, especially given that these terms do not translate well", so assuming CIB Resolution 2021-03 passes, I suggest that project takes this into account.

**Comment by heidivanparys** [comment]
> The background to this definition is most likely the ISO Directives (https://www.iso.org/sites/directives/current/consolidated/index.xhtml#_idTextAnchor415), which uses the phrase "Registration elements" which are defined as "Unique identifiers or identifier code components".

I just noticed that the introduction of ISO 19135-1 says:

> This part of ISO 19135 specifies procedures for the registration of items of geographic information. ISO/IEC JTC 1 defines registration as the assignment of an unambiguous name to an object in a way that makes the assignment available to interested parties.

So I double-checked what ISO/IEC JTC 1 actually states. They have different standards defining “registration” in different ways (of course…), but the one that was used for ISO 19135-1 (and earlier editions) is probably the one from [ISO/IEC 9834-1:2012,
_Information technology — Procedures for the operation of **object identifier registration** authorities: General procedures and top arcs of the international object identifier tree — Part 1_](https://www.iso.org/obp/ui#iso:std:iso-iec:9834:-1:ed-4:v1:en):

**registration**
The assignment of an unambiguous name to an object in a way which makes the assignment available to interested parties.
[ISO/IEC 9834-1:2012, term 3.5.16](https://www.iso.org/obp/ui#iso:std:iso-iec:9834:-1:ed-4:v1:en:term:3.5.16).

Given the title of the standard, in that context “registration” is the same as “object identifier registration”.

The “registration” concept that I need in my earlier work is closer to the following ISO/IEC JTC 1 concept:

**registration**
<metadata registry> inclusion of a metadata item in a metadata registry
[ISO/IEC 11179-1:2015, term 3.3.24](https://www.iso.org/obp/ui#iso:std:iso-iec:11179:-1:ed-3:v1:en:term:3.3.24)

**Comment by ronaldtse** [comment]
@heidivanparys we should also take into account the other terms related to "registration" in the latest 19135 draft.

It is then associated with the following terms:
* "addition": \<register> action taken to insert a new register item into a register (3.1.10)
* "register, noun": collection of register items
* "register item": \<register> description of information, including its identifier, for each item recorded in a register

The notion of the "inclusion" concept is similar to our "addition".

> So I would argue that the current definition is too narrow 

The project group considered a similar question in the previous meetings, namely: "how is a register different from a collection (or a set)"?

We came to the conclusion that a "register" provides each item with something that is "identifiable". This is the major difference with a generic collection.

In the case of the wedding register, the unique identifier (in the scope of the register) is the name of the entry. I would take it further in a "wedding registry" -- where guests could contribute to a wedding -- the whole point is to have the contributor names (uniquely) identifiable.

It is possible that "permanent-ness" is not an attribute of a registration, but the rest of the description seems fitting for an item inside a "register".

> and also against the principle that says that definitions must be application neutral 

We can apply a \<register> domain to it if you feel it is not generic enough?

With regards to the term "registry" causing issues in internationalisation, as you can see in the draft the preferred term is now "register system" (while "registry" is still admissible).

I'm including the other active participants that lead to the terminology so they can shed some light...

cc: @ReesePlews @maccraymer @rogerlott @Hothem @PJVorster

**Comment by PeterParslow** [comment]
I think it would be sensible to widen our current definition of registration to cover more than just assigning an identifier/name. I'm not sure about adopting the ISO/IEC 11179-1 definition either, not only because 'metadata' in our domain is more specific than I guess it is in theirs; to put it another way, using 'metadata' as we do, we register things that aren't metadata items.

I've noted that in the way the terms around registration are used in the UK in our domain, the emphasis is on a well managed list/catalogue - with governance that means a user can be reasonably sure that items in the register will persist (not 'for ever' necessarily) & be managed through a life cycle: i.e. open to suggestions of change, addition, superceding/retirement. It seems to me that the process / procedures of ISO 19135 are the important part - by saying that a register conforms to ISO 19135, one is saying that it is "well managed". I guess that's reflected in the title of the standard ("Procedures for ....")

So how about "registration" being defined as something like entering a data item into a register; where a register is something like a "well managed list (set, collection) of identified data items, managed in accordance with a defined set of procedures"?

I'm not putting these particular words forward as candidate definitions, just suggesting that either 'registration' or 'register' should make reference to the procedures/process/good management

**Comment by ronaldtse** [comment]
> I think it would be sensible to widen our current definition of registration to cover more than just assigning an identifier/name.

Agree with widening the definition.

> I'm not sure about adopting the ISO/IEC 11179-1 definition either, not only because 'metadata' in our domain is more specific than I guess it is in theirs; to put it another way, using 'metadata' as we do, we register things that aren't metadata items.

Indeed, we're not dealing with registration of metadata or a metadata register.

> So how about "registration" being defined as something like entering a data item into a register; where a register is something like a "well managed list (set, collection) of identified data items, managed in accordance with a defined set of procedures"?

I think this explanation is logical, and would support this change.

Considerations:
* I prefer the word "collection" instead of "list" as the latter conveys (at least in the computer science sense) it reflects an order of items
* We should refer back to the definition of "register, noun" which is simple, understandable and agreed across the project team in the last meeting

Proposal to update definitions:
* "register, noun" => "well managed collection of information items". (notice we need to define "well managed")
* "register item" => "<register> information item, including its identifier, recorded/contained in a register"
* "registration" => "<register> inclusion of a register item in a registry"
* and we need to define "well-managed"

Something like that?

This is more generic than what we had, and we no longer depend on a "unique and permanent identifier".

I think the project team needs to consider the following questions:
* Will a register item always be assigned an identifier within the register?
  * to me the answer is yes
* Will the register item always have an unambiguous identifier within the register?
  * to me this is a prerequisite of "well-managed"
* Will the register item always have a unique identifier within the register?
  * to me this is a prerequisite of unambiguous - two items with the same ID == ambiguous
* Will the register item always have a permanent identifier? Is the identifier not modifiable?
  * to me this is a no -- a permanent identifier is a huge ask. For example, we could still have a well managed register if we one day decide to alter the identification scheme (e.g. providing a mapping between old and new identifiers).

**Comment by PeterParslow** [comment]
I think it's better to say "persistent" than "permanent". In an (old but existing) UK government policy (on URI sets), "persistent" was explained as "intended to last ten years or more" - highlighting that good governance is a pre-requisite for persistent: ensuring that when things do change, they do so in a controlled way.

My candidate for "well managed": following the procedures given in ISO 19135?

**Comment by heidivanparys** [comment]
> > I'm not sure about adopting the ISO/IEC 11179-1 definition either, not only because 'metadata' in our domain is more specific than I guess it is in theirs; to put it another way, using 'metadata' as we do, we register things that aren't metadata items.
> 
> Indeed, we're not dealing with registration of metadata or a metadata register.

I agree with that, but there my point was more that I would make that definition, “inclusion of a metadata item in a metadata registry”, application neutral, then I would get “inclusion of an item in a registry”, which is close to the definition I ended up having in my other project, “the act of entering something into an information system”.


> The project group considered a similar question in the previous meetings, namely: "how is a register different from a collection (or a set)"?
> 
> We came to the conclusion that a "register" provides each item with something that is "identifiable". This is the major difference with a generic collection.

> * "register, noun" => "well managed collection of information items". (notice we need to define "well managed")

> My candidate for "well managed": following the procedures given in ISO 19135?

Personally, I like the proposal of using “well managed” in the definition of register. This is a bit broader than “formal” or “official”, which can be found in many [dictionaries](https://www.thefreedictionary.com/register). Regarding having to define it: an alternative would be to add a note, explaining the meaning of “well managed” with more words. I would not say “following the procedures given in ISO 19135”, because that would make the definition/note not application neutral anymore, other decent procedures exist, I suppose.

Some inspiration:
- well-managed = used to describe a business, economy, etc. that is controlled in a way that produces good results (https://dictionary.cambridge.org/us/dictionary/english/well-managed)
- administered or controlled in a competent or successful manner (https://www.collinsdictionary.com/dictionary/english/well-managed)

**Comment by ronaldtse** [comment]
> I would not say “following the procedures given in ISO 19135”, because that would make the definition/note not application neutral anymore, other decent procedures exist, I suppose.

Agree. Perhaps the word "managed" is less controversial than "well managed", since the former still implies that someone is managing it, with no prejudice on the outcome.

**Comment by PeterParslow** [comment]
Just for info, I didn't seriously expect us to require 19135.

How about something like "managed", with a note "managed using a well defined procedure, such as the one defined in this document".
